.github/topissuebot.yml

```diff
@@ -0,0 +1,4 @@
+# Configuration for https://probot.github.io/apps/topissues/
+labelName: ":thumbsup: Top Issue!"
+labelColor: "f442c2"
+numberOfIssuesToLabel: 50

```

Comment:
Do we want this many? That's two pages on GitHub 🤔 - here's the current list of most :+1:'d issues; https://github.com/moby/moby/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc